lib/aba/batch.rb

```diff
@@ -193,7 +194,7 @@ def batch_control_record
       # Net total
       # Max: 10
       # Char position: 21-30
-      output += net_total_amount.abs.to_s.rjust(10, "0")
+      output += (credit_total_amount - debit_total_amount).abs.to_s.rjust(10, "0")
```

Comment:
I don't think `abs` is required here anymore.
```suggestion
      output += (credit_total_amount - debit_total_amount).to_s.rjust(10, "0")
```

Reply:
Actually this has to stay in case the `debit_total_amount` is larger than the `credit_total_amount`

Reply:
Yes that makes sense, now you say it. 
Is there a spec to highlight the larger debit than credit total behaviour?